### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by j-tesla**
Add the `requests` module

fixes kossiitkgp#15

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -19,3 +19,4 @@ black
 bandit
 flake8
 pre-commit
+requirements
```
Comment (rakaar) [inline requirements.txt:22]: it should be requests right?

**Review by rakaar — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (j-tesla) [reply requirements.txt:22]: extremely sorry! my bad!

**Comment by rakaar** [comment]
Hey @j-tesla 
I tried forking and added the PAT too, but there were no new commits in my fork. You can check [here](https://github.com/rakaar/kwoc-stats-api/tree/j-tesla-stats-workflow-patch)

I even tried changing the time interval to for every 2 mins, u can check it [here](https://github.com/rakaar/kwoc-stats-api/commit/548b76221968a7311d9d5b433dbeae2fc39149f2)

What am I missing?
Can u please help me out

**Review by rakaar — CHANGES_REQUESTED** [review]
sorry for telling it late
but for packages that are utilized in production, it is better to have version number
actually it is good to have a version for all packages. But I didn't notice it, my bad!

Please add the latest version of requests - as [here](https://pypi.org/project/requests/)
if u are free, u can add the version for other packages too. It would a great help

**Comment by rakaar** [comment]
thank you so much @j-tesla  for the patience in  adding version numbers for other requirements too!

**Pull request merged**